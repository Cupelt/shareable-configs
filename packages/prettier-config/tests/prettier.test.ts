import prettierConfig from '../src'

describe('Prettier Config', () => {
  test('should export rules', () => {
    expect(prettierConfig.$schema).toBe('http://json.schemastore.org/prettierrc')
    expect(prettierConfig.useTabs).toBe(false)
    expect(prettierConfig).toMatchSnapshot()
  })
})
