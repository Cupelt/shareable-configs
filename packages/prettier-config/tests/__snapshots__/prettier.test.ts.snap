// Vitest Snapshot v1

exports[`Prettier Config > should export rules 1`] = `
{
  "$schema": "http://json.schemastore.org/prettierrc",
  "arrowParens": "avoid",
  "bracketSpacing": true,
  "endOfLine": "lf",
  "overrides": [
    {
      "files": ".all-contributorsrc",
      "options": {
        "parser": "json",
      },
    },
  ],
  "printWidth": 100,
  "quoteProps": "as-needed",
  "semi": false,
  "singleQuote": true,
  "trailingComma": "none",
  "useTabs": false,
}
`;
